### DISCUSSION TITLE
I can't move using (HJKL) sometimes

### DISCUSSION THREAD

**Discussion by carlinhoshk** [JetBrains/ideavim#991] [Q&A] [answered]
Why can't I sometimes move between lines even when I use KJ? It always gets stuck after I hit an empty line, so I have to exit normal mode and go back to move within the insert using the arrow keys.
Have I reached some usage limit to learn how to move using lines by numbers, for example (10 + J)?

![image](https://github.com/user-attachments/assets/b6ba7df8-886f-4a37-a5d6-7640d57762ad)

**Comment by citizenmatt** [comment, marked as answer]
What's the message saying "⬆️L to Chat, I for command"? I presume it's some kind of plugin - perhaps this is clashing with IdeaVim's motions?

**Reply by carlinhoshk** [reply]
Thanks, that was it, another plugin in this case (codeium copilot)
